producer.py

```diff
@@ -1,11 +1,34 @@
-from json_asset import scrape
+from scraper import scrape, current_time
+from area_dict import area
 
-def producer():
-    raw_json = scrape('C03')
+current_time = current_time()
 
-    events = {}
+def producer(): #iterating each area in area_dict and returns a dictionary of all areas
+    all_areas = {}
+
+    for area_name in area.values():
+        area_code = area[area_name]
+        all_areas[area_code] = json_rearranger(area_code)
+    
+    return current_time, all_areas
+
+def json_rearranger(area_code): #fetch json and rearrange each element with an updated key
+    raw_json = scrape(area_code, current_time)
```

Comment:
Having the function to read global variable are not recommended unless they are constant or hyperparameters. 
The variable `current_time` is generated by another function and declared in global scope, which brings uncertainty to your output.

Reply:
Makes sense, then calling `current_time` somewhere only once and then passing the value as a prop to other functions is better in order to keep the value as it is at that moment, I guess?

Reply:
Yes, or if `current_time` will be update whenever the function is called, you may simply generate `current_time` in the function scope too.